UIKitDZ/UIKitDZ/ViewController.swift

```diff
@@ -5,9 +5,188 @@ import UIKit
 
 /// ViewController
 class ViewController: UIViewController {
-//    override func viewDidLoad() {
-//        super.viewDidLoad()
+    lazy var guessButton: UIButton = {
+        let button = UIButton(frame: CGRect(x: 85, y: 302, width: 150, height: 150))
+        button.setTitle("Угадай число", for: .normal)
+        button.titleLabel?.font = UIFont(name: "Verdana-Bold", size: 20)
+        button.titleLabel?.lineBreakMode = .byWordWrapping
+        button.titleLabel?.textAlignment = .center
+        button.setTitleColor(.white, for: .normal)
+        button.layer.cornerRadius = CGFloat(30)
+        button.backgroundColor = .systemPurple
+        button.layer.borderColor = .init(red: 76 / 255, green: 36 / 255, blue: 115 / 255, alpha: 1)
+        button.addTarget(self, action: #selector(guessNumberButtonTapped), for: .touchUpInside)
+        button.layer.borderWidth = 2
+        return button
+    }()
 
-//        // Do any additional setup after loading the view.
-//    }
+    lazy var calculateButton: UIButton = {
+        let button = UIButton(frame: CGRect(x: 140, y: 510, width: 200, height: 200))
+        button.setTitle("Калькулятор", for: .normal)
+        button.titleLabel?.font = UIFont(name: "Verdana-Bold", size: 20)
+        button.setTitleColor(.white, for: .normal)
+        button.layer.cornerRadius = CGFloat(30)
+        button.backgroundColor = .green
+        button.backgroundColor = .systemGreen
+        button.layer.borderColor = .init(red: 59 / 255, green: 105 / 255, blue: 76 / 255, alpha: 1)
+        button.layer.borderWidth = 2
+        button.addTarget(self, action: #selector(additionButtonTapped), for: .touchUpInside)
+        return button
+    }()
+
+    lazy var guessLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 0, y: 48, width: Int(view.frame.width), height: 100))
+        label.font = UIFont(name: "Verdana-Bold", size: 30)
+        label.textAlignment = .center
+        label.textColor = .white
+        label.backgroundColor = UIColor(red: 151 / 255, green: 203 / 255, blue: 255 / 255, alpha: 1)
+        label.layer.borderColor = CGColor(red: 255, green: 255, blue: 255, alpha: 1)
+        label.layer.borderWidth = 2.0
+        label.numberOfLines = 0
+        label.isHidden = true
+        return label
+    }()
+
+    lazy var backgroundImage: UIImageView = {
```

Comment:
```suggestion
    private lazy var backgroundImage: UIImageView = {
```
Дополнительно - название вью не соответствует типу. Если тип - imageView, то название должно заканчиваться на imageView.

```suggestion
    private lazy var backgroundImageView: UIImageView = {
```